book_class.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
![image](https://github.com/mahammad-mostafa/school-library/assets/78251927/9b650c26-1b36-4d6d-a9c3-da048a8cb09a)

Getters and Setters for @rentals should be present